src/containers/Projects/ProjectRulesAndGuide/index.tsx

```diff
@@ -1,54 +1,72 @@
+import React, {FC} from 'react';
+import {Link} from 'react-scroll';
 
-import {DashboardLayout, PageTitle, Pagination} from 'components';
-import {PageData, PageDataObject} from 'types/page-data';
+import HowProposalsWork from './HowProposalsWork';
+import Rules from './Rules';
+import ProposalSubmissionProcess from './ProposalSubmissionProcess';
+import MilestonesAndPayouts from './MilestonesAndPayouts';
+import './ProjectRules.scss';
 
-import ProjectsMenuItems, {projectsNavigationData} from '../ProjectsMenuItems';
-import ProjectsMilestones from './ProjectsMilestones';
-import ProjectsOverview from './ProjectsOverview';
-import ProjectsProposalSubmissionProcess from './ProjectsProposalSubmissionProcess';
-import ProjectsRules from './ProjectsRules';
+interface Section {
+  id: string;
+  title: string;
+}
 
-const defaultPageData: PageData = {
-  content: <Redirect to="/project-rules/overview" />,
-  name: '',
-};
-
-const pageData: PageDataObject = {
-  milestones: {
-    content: <ProjectsMilestones />,
-    name: 'Milestones & Payouts',
-  },
-  overview: {
-    content: <ProjectsOverview />,
-    name: 'Overview',
-  },
-  proposals: {
-    content: <ProjectsProposalSubmissionProcess />,
-    name: 'Proposal Submission Process',
-  },
-  rules: {
-    content: <ProjectsRules />,
-    name: 'Rules & Guidelines',
-  },
-};
-
-const getPageData = (chapter: string): PageData => {
-  return pageData[chapter] || defaultPageData;
-};
-
-const Projects: FC = () => {
-  const {chapter} = useParams<{chapter: string}>();
-  const {content, name} = useMemo(() => getPageData(chapter), [chapter]);
+const ProjectRules: FC = () => {
+  const SECTIONS: Section[] = [
+    {
+      id: 'how-proposals-work',
+      title: 'How Proposals Work',
+    },
+    {
+      id: 'rules',
+      title: 'Rules',
+    },
+    {
+      id: 'proposal-submission-process',
+      title: 'Submission Process',
+    },
+    {
+      id: 'milestones-and-payouts',
+      title: 'Milestones & Payouts',
+    },
+  ];
 
   return (
-    <DashboardLayout menuItems={<ProjectsMenuItems />} pageName={name} sectionName="Projects">
-      <PageTitle ogDescription={`${name} | Projects`} title={name} />
-      {content}
-      <Pagination navigationData={projectsNavigationData} />
-    </DashboardLayout>
+    <div className="ProjectRules">
+      <header className="ProjectRules__banner">
+        <h1 className="ProjectRules__banner-headline">Project Rules and Guidelines</h1>
+        <p className="ProjectRules__banner-text">The detailed guidelines about how the projects works. </p>
+      </header>
+      <main className="ProjectRules__main">
+        <aside className="ProjectRules__sidebar">
+          {SECTIONS.map((section) => (
+            <Link
+              activeClass="ProjectRules__sidebar-item-active"
+              className="ProjectRules__sidebar-item"
+              hashSpy
+              key={section.id}
+              offset={-308}
```

Comment:
perhaps can refactor this as a constant and state that it is TOP_NAV_HEIGHT + HEADER_HEIGHT?